packages/chain-agnostic-permission/src/adapters/caip-permission-adapter-accounts.test.ts

```diff
@@ -3,9 +3,14 @@ import type { CaipAccountId } from '@metamask/utils';
 import {
```

Comment:
Doesn't have to be in this PR, but we should rename this `adapter` files

Reply:
yeah, let's do it in a follow up PR, but totally agree